github_utils.py

```diff
@@ -93,6 +93,8 @@ def get_ported_PRs(repo, src_branch, des_branch):
     prRe = re.compile('Automatically ported from ' + src_branch + ' #(\d+)\s+.*', re.MULTILINE)
     for pr in repo.get_pulls(base=des_branch):
         body = pr.body.encode("ascii", "ignore")
+        if sys.version_info[0] == 3:
```

Comment:
This function, and `def port_pr`, are only used by `port-pull-request` script, which is not referenced by any job.